build/scripts/cleanup_perf.ps1

```diff
@@ -25,13 +31,16 @@ mv C:/CPC/DataBackup* ToArchive
 ls ToArchive
 
 # Clean CPC and related directories out of the machine
+$ExitCode = 0
 If (Test-Path C:/CPC) {
     try {
         echo "removing C:/CPC ..."
         Remove-Item -Recurse -Force C:/CPC
         echo "done."
     }
-    catch { }
+    catch {
+        $ExitCode = 1
```

Comment:
Yes, this is intended.  The thought was "do as much cleanup as possible, and then if any of them failed, report failure"